scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -0,0 +1,556 @@
+            instance_name = self.DEFAULT_CLOUDSQL_INSTANCE_NAME
+        self.cloudsql_instance = instance_name
+
+    def setup_cloudsql_user(self):
+        """Ask user to input Cloud SQL user name."""
+        sql_user = raw_input(
+            'Enter the sql user name of your choice '
+            '(default: {}) '
+            .format(self.DEFAULT_CLOUDSQL_USER)).strip().lower()
+        if not sql_user:
+            sql_user = self.DEFAULT_CLOUDSQL_USER
+        self.cloudsql_user = sql_user
+
+    def create_cloudsql_instance(self):
+        """Create Cloud SQL instance."""
+        if self.cloudsql_instance:
+            print('Creating Cloud SQL instance... This will take awhile...')
+
+            proc = subprocess.Popen(
+                ['gcloud', 'sql', 'instances', 'create', self.cloudsql_instance,
+                 '--database-version=%s' % self.CLOUDSQL_DB_VERSION,
+                 '--tier=%s' % self.CLOUDSQL_TIER,
+                 '--storage-size=%s' % self.CLOUDSQL_STORAGE_SIZE_GB,
+                 '--storage-type=%s' % self.CLOUDSQL_STORAGE_TYPE],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE)
+            _, err = proc.communicate()
+            if proc.returncode != 0:
+                print(err)
+
+        if self.cloudsql_instance:
+            proc = subprocess.Popen(
+                ['gcloud', 'sql', 'users', 'create',
+                 self.cloudsql_user, 'localhost',
+                 '--instance', self.cloudsql_instance],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE)
+            _, err = proc.communicate()
+            if proc.returncode != 0:
+                print(err)
+
+    def generate_deployment_templates(self):
+        """Generate deployment templates."""
+        # TODO: generate deployment templates
+        # TODO: ask if we should create the deployment
+        pass
+
+    def create_data_storage(self):
+        """Create Cloud SQL instance and Cloud Storage bucket. (optional)"""
+        self._print_banner('Create data storage (optional)')
+        should_create_bucket = raw_input(
+            'Create Cloud Storage bucket now? (y/N) ').strip().lower()
+        if should_create_bucket == 'y':
+            self.create_bucket()
+        should_create_cloudsql = raw_input(
+            'Create Cloud SQL instance now? '
+            'This will take awhile. (y/N) ').strip().lower()
+        if should_create_cloudsql == 'y':
+            self.create_cloudsql_instance()
+
+    def post_install_instructions(self):
+        """Show post-install instructions.
+
+        Print link to go to GSuite service account and enable DWD, if GSuite
+        service account was created.
+        Print link for deployment manager dashboard.
+        """
+        self._print_banner('Post-setup instructions')
+
+        print('Congratulations! You\'ve completed the Forseti prerequisite '
+              'setup.')
+
+        if self.gsuite_service_acct:
```

Comment:
There's additional steps for them to enable the service-account and dwd-id in the GSuite control panel. I don't think I saw [those](http://forseti-security.io/service_accounts/) here.